scripts/screens/cat_screens.py

```diff
@@ -2357,10 +2356,11 @@ def update_selected_cat(self):
         )
 
         name = str(self.the_cat.name)
-        if len(name) > 17:
-            name = name[:14] + "..."
-        self.selected_cat_elements["cat_name"] = pygame_gui.elements.UILabel(scale(pygame.Rect((790, 140), (320, -1))),
-                                                                             name,
+        length_limit = 300
+        font_size = 26
+        short_name = shorten_text_to_fit(name, length_limit, font_size)
```

Comment:
I suggest removing the variables "length_limit" and "font_size", and simple pass these values directly into shorten_text_to_fix.  These variables are not used outside of the function call, and removing unnecessary variables increased readability.